docs/tutorialkit.dev/src/content/docs/reference/theming.mdx

```diff
@@ -316,6 +316,17 @@ The navigation cards are the cards at the bottom of a lesson to navigate to the
 | `--tk-elements-navCard-iconColor`            | The icon color of the navigation card.                     |
 | `--tk-elements-navCard-iconColorHover`       | The icon color of the navigation card when hovering.       |
 
+### Webcontainers link
+
+Link to [webcontainers.io](https://webcontainers.io/) that's shown below navigation cards.
+
+![Webcontainers Link](./images/theming-webcontainers.png)
+
+| Token                                            | Description                                         |
+| ------------------------------------------------ | --------------------------------------------------- |
+| `--tk-elements-webcontainersLink-textColor`      | The text color of the link                          |
+| `--tk-elements-webcontainersLink-textColorHover` | The text color of the link when hovering.           |
```

Comment:
I just realised that we're introducing tokens for this. Would it maybe make sense to use a more generic set of tokens for links? Maybe we have some already?

Reply:
I wasn't so sure about these either - generic tokens would be great.

There is already `--tk-elements-app-linkColor` but the style doesn't match with this link's design.

Reply:
What about links in markdown? What tokens do they use?

Reply:
They use `--link-color: var(--tk-text-accent);` which is same as `--tk-elements-app-linkColor` I think.

Reply:
Ah so maybe we could change them to use a new `--tk-elements-linkColor` token. WDYT?

By the way for the `Powered by WebContainer` we could also use the same tokens as for link but dim it a bit by using an `opacity-50` class.

Reply:
> Ah so maybe we could change them to use a new `--tk-elements-linkColor` token. WDYT?

Sure, I'll apply the changes here:

https://github.com/stackblitz/tutorialkit/blob/cc50ad4d90d6ffb42f3a658de08f9617826b6c07/packages/astro/src/default/styles/markdown.css#L12

> By the way for the `Powered by WebContainer` we could also use the same tokens as for link but dim it a bit by using an `opacity-50` class.

You mean use same `--tk-elements-linkColor` with `opacity-50`?

<img src="https://github.com/user-attachments/assets/b18a5188-7bfc-4545-a75d-6a6b9ed34a07" height="300" />

Reply:
Yes that's what I meant! 

However, now that you shared on it looks, maybe we still want a secondary link so that the colour is gray instead of the accent color. Your previous version was better. The opacity would still be useful to dim it more (as we probably want that one to be more discreet than secondary links if we ever add others).

Reply:
Alright this is now implemented in https://github.com/stackblitz/tutorialkit/pull/202/commits/c6c267fca359985d1d83afd540f5354cb0b3648d. Let me know what you think, I wasn't really sure if this is how it was intended. There's also `TODO What here?` in link's documentation that needs to be resolved. Also removing all `--tk-elements-editPageLink-*` is a breaking change so need to mark that before merging.